spec/tapioca/dsl/compilers/active_record_columns_spec.rb

```diff
@@ -356,6 +356,61 @@ def string_enum_column=(value); end
                 assert_includes(output, expected)
               end
 
+              it "generates correct types for serialized columns" do
```

Comment:
I moved this out to separate test case, since the other one was getting a little busy.